sources/src/resources/init-scripts/gradle-actions.inject-develocity.init.gradle

```
@@ -21,6 +21,8 @@ initscript {
     }
 
     def pluginRepositoryUrl = getInputParam('gradle.plugin-repository.url')
+    def pluginRepoUsername = getInputParam('gradle.plugin-repository.username')
+    def pluginRepoPassword = getInputParam('gradle.plugin-repository.password')
```

I would spell out `repository` so it is the same as the existing `pluginRepositoryUrl`.

```suggestion
    def pluginRepositoryUsername = getInputParam('gradle.plugin-repository.username')
    def pluginRepositoryPassword = getInputParam('gradle.plugin-repository.password')
```